### DISCUSSION TITLE
Dashboard for Grafana 8 alerts: Ability to show only firing and pending instances for an alert

### DISCUSSION THREAD

**Discussion by Dhyanesh97** [grafana/grafana#39833] [Dashboards, Panels & Visualizations]
![image](https://user-images.githubusercontent.com/89400147/135412776-91159391-9069-4f0f-a750-57d862bdb14e.png)

The instances which are shown for the firing alerts should only display records with Alerting or Pending state. We should get ability to specify which records/instance should be displayed on the dashboard. (In my case i only want alerting and pending instance)

Currently it shows records for all the instance(Alerting, Normal, Pending). User needs to scroll down to get the details of the instance in Alerting state on dashboard.

Also if we change the start time and end time on the dashboard, alerts should be displayed as per time and the state in which they were in that time frame.

**Comment by Dhyanesh97** [comment]
![image](https://user-images.githubusercontent.com/89400147/135460511-6a5c3ed7-4895-4904-9b73-6e9646986098.png)

Here instead of showing all the states in the list of instances, we only want to show the alerting instance.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
